libraries/CMakeLists.txt

```diff
@@ -45,3 +45,10 @@ set_property(GLOBAL PROPERTY CTEST_CUSTOM_TESTS_IGNORE
   attest pbkdf2 parsing ${_CTEST_CUSTOM_TESTS_IGNORE}")
 
 add_subdirectory( rodeos )
+
+message(STATUS "Starting AMQP-CPP configuration...")
```

Comment:
it's not really a problem to have this Starting line and the accompanying Complete line. But, the reason it was done for yubihsm in the same file (which I'm guessing you took as reference) is because yubihsm spews a lot of somewhat confusing messages to the cmake log. amqp-cpp seems rather silent so it probably doesn't _need_ these messages. it's not critical to remove though, you can decide